docs/source_values.rst

```diff
@@ -38,12 +38,17 @@ GPIO Zero provides a method of using the declarative programming paradigm to
 connect devices together: feeding the values of one device into another, for
 example the values of a button into an LED:
 
+.. image:: images/source_values/led_button.*
+    :align: center
+
 .. literalinclude:: examples/led_button.py
 
 which is equivalent to:
 
 .. literalinclude:: examples/led_button_loop.py
 
+except that the former is updated in a background thread.
+
 Every device has a :attr:`~Device.value` property (the device's current value).
 Input devices can only have their values read, but output devices can also have
```

Comment:
This might be clearer as "Input devices (like buttons) can only have their values read, but output devices (like LEDs) can also ..." ?

Reply:
Done - in next push